packages/block-editor/src/components/block-navigation/index.js

```diff
@@ -16,13 +15,41 @@ import { __ } from '@wordpress/i18n';
 import BlockNavigationTree from './tree';
 import { store as blockEditorStore } from '../../store';
 
-function BlockNavigation( {
-	rootBlock,
-	rootBlocks,
-	selectedBlockClientId,
-	selectBlock,
+export default function BlockNavigation( {
+	onSelect = noop,
 	__experimentalFeatures,
 } ) {
+	const { rootBlock, rootBlocks, selectedBlockClientId } = useSelect(
+		( select ) => {
+			const {
+				getBlockHierarchyRootClientId,
+				getSelectedBlockClientId,
+				__unstableGetClientIdsTree,
+				__unstableGetClientIdWithClientIdsTree,
+			} = select( blockEditorStore );
+
+			const _selectedBlockClientId = getSelectedBlockClientId();
+			const _rootBlocks = __unstableGetClientIdsTree();
+			const _rootBlock = _selectedBlockClientId
```

Comment:
It's because those names are already declared in the upper scope.
The `_` is just simpler than figure out some naming switcheroo that might compromise the readability.

(Also, I've seen it used plenty of times in Gutenberg, sometimes to mark a constant as "private", but often in this exact same context of select hooks.)